packages/core/src/cli/mf.ts

```diff
@@ -25,8 +25,15 @@ async function initMFRsbuild(
   }
 
   mfRsbuildConfig.config = changeEnvToDev(mfRsbuildConfig.config);
+
   const rsbuildInstance = await createRsbuild({
-    rsbuildConfig: mfRsbuildConfig.config,
+    rsbuildConfig: {
+      ...mfRsbuildConfig.config,
+      plugins: [
+        ...(rslibConfig.plugins || []),
+        ...(mfRsbuildConfig.config.plugins || []),
```

Comment:
Why `mfRsbuildConfig.config.plugins` is added to shared plugins?

Reply:
This is not the shared plugins, it's the merged final plugins

Reply:
Got it, we should refactor this to using `environments` as same as `build` and `inspect` command.